### DISCUSSION TITLE
Drybox ideas

### DISCUSSION THREAD

**Discussion by TurboTronix** [Enraged-Rabbit-Community/ERCF_v2#142] [General]
Does anyone have a cool drybox idea to share for the 8 spools version? I got my Ender 3 in an Ikea Lack installation and was wondering if there was a way to add a drybox around the spools.

**Comment by bigtalljv** [comment]
I remixed the Prusa (un)original drybox to fit 8 filametrix rewinders.  My CAD skills are weak, so the files are a bit rough 
 https://www.printables.com/model/1157072-remixed-unoriginal-prusa-heated-drybox
